docs/src/main/sphinx/release/release-408.md

```diff
@@ -41,6 +43,11 @@
 * Fix performance regression when writing to partitioned tables if table
   statistics are absent. ({issue}`16152`)
 
+## Example HTTP connector
```

Comment:
I don't think we need to document this, since the connector isn't documented for end users to use directly -- it's there to be example code.